common.gypi

```diff
@@ -10,7 +10,7 @@
       # compiling your module on any ubuntu version, even the latest
       # where the CXX11_ABI default has flipped to 1
       # More details at https://github.com/mapbox/mason/issues/319
-      '-D_GLIBCXX_USE_CXX11_ABI=0'
+      # '-D_GLIBCXX_USE_CXX11_ABI=0'
```

Comment:
Some default has changed with all the versions bumps, so if this flag is set (=0) it will be set two times as 0 and once (from some default I suppose) as 1, which causes error. 

Removing this solves the issue, but I suppose we should find out if this flag is necessary for some reason.